src/gui/guiFormSpecMenu.cpp

```diff
@@ -1645,19 +1661,33 @@ void GUIFormSpecMenu::parseLabel(parserData* data, const std::string &element)
 
 			if (data->real_coordinates) {
 				// Lines are spaced at the distance of 1/2 imgsize.
-				// This alows lines that line up with the new elements
-				// easily without sacrificing good line distance.  If
+				// This allows lines that line up with the new elements
+				// easily without sacrificing good line distance. If
 				// it was one whole imgsize, it would have too much
 				// spacing.
 				v2s32 pos = getRealCoordinateBasePos(v_pos);
 
-				// Labels are positioned by their center, not their top.
-				pos.Y += (((float) imgsize.Y) / -2) + (((float) imgsize.Y) * i / 2);
+				// Add newline spacing
+				pos.Y += (((float) imgsize.Y) * stof(style.get(StyleSpec::NEWLINE_SPACING, "0.5")) * i);
```

Comment:
Line needs breaking. Also superfluous parentheses and space in `((float) imgsize.Y)`.